lib/Haptics.ts

```diff
@@ -0,0 +1,93 @@
+import * as ExpoHaptics from "expo-haptics"
+import { createDependencyKey } from "./dependencies"
+
+/**
+ * A haptic event to be played to the user.
```

Comment:
Nice, what kind of haptics will we have

Reply:
Rn it just delegates to `expo-haptics`, but on iOS in particular (as long as we support iOS 13+), we can actually add very customized/personalized haptics (which includes adding sound effects) using a framework called [CoreHaptics](https://developer.apple.com/documentation/corehaptics).

Unfortunately, I don't think there's a well maintained react native library that wraps this framework, so we would have to introduce native code if we wanted to use it.